client/src/Components/Header.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import useDarkMode from "../Helpers/useDarkMode";
+
+export default class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      darkMode: false
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  componentDidUpdate() {
+    document.addEventListener("click", this.toggle);
+    useDarkMode(this.state.darkMode);
```

Comment:
From this [link](https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both), hooks is not supposed to be used inside class components